src/parse_logcfg.h

```diff
@@ -109,10 +109,6 @@ typedef struct {
         (cfg_arg_t){.char_pp=&var, .chomp=true, \
                     .string_type=DYNAMIC}
 
-#define CFG_MESSAGE(var, i)     NEED_PARAM, cfg_string, \
```

Comment:
Maybe we could keep CFG_MESSAGE name instead the CHOMP. (as there is no need to distinguish between the two)

Reply:
Yes, good idea.

Reply:
done. see commit below